assets/js/blocks/cart-checkout/cart-i2/edit.tsx

```diff
@@ -181,7 +127,11 @@ const CartEditor = ( { className, attributes, setAttributes } ) => {
 	} );
 	const defaultInnerBlocksTemplate = [
 		[ 'woocommerce/cart-items-block', {}, [] ],
-		[ 'woocommerce/cart-totals-block', {}, [] ],
+		[
+			'woocommerce/cart-totals-block',
+			{},
+			[ [ 'woocommerce/cart-order-summary-block', {}, [] ] ],
```

Comment:
I didn't add `showRateAfterTaxName` here yet, it's for Cart i1 to Cart i2 compatibility.